requirements.txt

```diff
@@ -1,2 +1,3 @@
 SQLAlchemy==2.0.0
-python-redmine
+ratelimiter==1.2.0.post0
+requests==2.28.2
```

Comment:
I would rather have `equal or bigger` assignment here. is there any reason to have fixed versions?